packages/loopring_v3/contracts/test/AmmPool.sol

```diff
@@ -0,0 +1,465 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2017 Loopring Technology Limited.
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "../aux/access/IBlockReceiver.sol";
+import "../aux/transactions/BlockReader.sol";
+import "../thirdparty/BytesUtil.sol";
+import "../core/iface/IExchangeV3.sol";
+import "../lib/AddressUtil.sol";
+import "../lib/ERC20.sol";
+import "../lib/ERC20SafeTransfer.sol";
+import "../lib/MathUint.sol";
+
+import "../core/impl/libtransactions/AmmUpdateTransaction.sol";
+import "../core/impl/libtransactions/DepositTransaction.sol";
+import "../core/impl/libtransactions/WithdrawTransaction.sol";
+
+contract AmmPool is IBlockReceiver {
```

Comment:
@Brechtpd please just add a simple comment saying that this contract is not fully implemented and serves as a demo of how AMM pool may be created on top of Loopring.